src/styles/_normalize.scss

```diff
@@ -0,0 +1,25 @@
+html {
+  font-family: $page-font;
+  text-decoration: none;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+
+  * {
```

Comment:
why is it inside body?

Reply:
> why is it inside body?

```
  * {
     display: flex;
  }
```
makes <title> visible on page if rule isn't nested in body